doc/input-output-reference/src/input-for-output.tex

```diff
@@ -518,6 +515,12 @@ \subsubsection{Field: Name}\label{field-name-1-036}
 \endhead
 
 EnergyTransfer \tabularnewline
+$cdots$ \tabularnewline
```

Comment:
This line renders as 
![image](https://github.com/NREL/EnergyPlus/assets/4991075/e2e13039-f1b0-44b2-a5b6-69fa08210d2d)

Reply:
A typo. Should be $\cdots$ instead to render correctly.